### ISSUE TITLE
Demo page 404

### ISSUE THREAD

**Issue by hennadiy-kudashev** [insin/nwb#542]
This issue is a: 
- Bug report

Thanks for 0.24.0 release. I have upgraded and have error with loading demo page.

Steps to reproduced (on new setup):
- nwb new react-component nwb-component
- yarn add nwb //upgrade to 0.24.0
- yarn start

Demo page has not loaded 404. Also have tried with webpack.html.template - the same issue.

nwb: 0.24.0
node: 11.10.1
yarn: 1.16.0

**Comment by insin** [comment]
Thanks for the bug report - fixed in v0.24.1

**Comment by insin** [comment]
Protip, nwb's config is designed to work globally or locally, so you don't have to immediately add it to new projects to use it 😸 

```console
$ nwb new react-component react-component -f
Creating a react-component project...
  create .gitignore
  create .travis.yml
  create CONTRIBUTING.md
  create README.md
  create demo\src\index.js
  create nwb.config.js
  create package.json
  create src\index.js
  create tests\.eslintrc
  create tests\index.test.js
√ Installing react and react-dom
√ Initing Git repo

λ cd react-component\

λ npm start -- --no-clear

> react-component@1.0.0 start C:\Users\Jonny\repos\nwb-manual-tests\react-component
> nwb serve-react-demo "--no-clear"

Starting Webpack compilation...
i ｢wds｣: Project is running at http://localhost:3000/
i ｢wds｣: webpack output is served from /
i ｢wds｣: Content not from webpack is served from C:\Users\Jonny\repos\nwb-manual-tests\react-component
i ｢wds｣: 404s will fallback to /index.html
Compiled successfully in 1709 ms.

The app is running at http://localhost:3000/
```

(I really need to do something about that new `[wds]` logging.
